extension/lib/magento/customer/getDiff.js

```diff
@@ -0,0 +1,37 @@
+const { differenceWith, isEqual } = require('lodash')
```

Comment:
You could also do something like this:
`const _ = {
  isEqual: require('lodash/isEqual'),
  differenceWith: require('lodash/differenceWith')
}`

and later us it with "\_.differenceWith" and "\_.isEqual". So you know that those methods are taken from lodash, just by reading the code.